server/package.json

```diff
@@ -12,11 +12,13 @@
   "dependencies": {
     "bcrypt": "^3.0.6",
     "body-parser": "^1.19.0",
+    "bottleneck": "^2.19.5",
```

Comment:
Let's remove this dependency if it's no longer something we need.

Reply:
done~